src/core_plugins/metrics/server/lib/vis_data/helpers/bucket_transform.js

```diff
@@ -1,5 +1,7 @@
 import parseSettings from './parse_settings';
 import getBucketsPath from './get_buckets_path';
+import toMS from './to_milliseconds';
```

Comment:
Instead of using this we should use `elastic/datemath-js` since it's already part of the project and we use that in other places.

Reply:
@simianhacker I'm not sure how you want me to use `elastic/datemath`. Its only exposed function is `parse()` which doesn't take a time interval as input.

I took the `to_milliseconds` module from the `timelion` plugin, but I'm happy to exchange it with something else that can convert a time interval `1m`, `1h`, etc to milliseconds.

Reply:
Doah... sorry... I thought datemath had a interval parse function. How about this: https://github.com/elastic/kibana/blob/master/src/ui/public/utils/parse_interval.js You can include it using `import { parseInterval } from 'ui/utils/parse_interval';` That will return a Moment duration then you can just convert it to milliseconds.

Reply:
@simianhacker done + rebased. I also added a text similar to the one on math aggregations to let users know about the new functionality.